drivers/windows/R/web.R

```diff
@@ -109,11 +109,12 @@ web_client <- R6::R6Class(
       client_parse_r_versions(httr_text(r))
     },
 
-    cluster_resources = function(cluster, driver) {
-      r <- private$client$GET(
-        sprintf("/api/v1/cluster_info/%s/%s", cluster, driver),
-        public = TRUE)
-      client_parse_cluster_resources(httr_text(r))
+    cluster_resources = function() {
+      r_windows <- private$client$GET(
+        "/api/v1/cluster_info/wpia-hn/hipercow.windows/", public = TRUE)
```

Comment:
These names are immediately out of date as they look like package names (we had `hipercow.windows` but now we are working towards `hipercow.dide`.  I know that the portal will be replaced by a new API soon but we might want to not bake in these names all the same

Reply:
Maybe `windows` and `linux` are fine here, to match config$platform

Reply:
I've updated to use windows and linux